From 2a732bb03812878a9cc00d27d6c80f3993520626 Mon Sep 17 00:00:00 2001
From: Petr Mensik <pemensik@redhat.com>
Date: Nov 26 2020 16:33:42 +0000
Subject: Support BIND 9.16.9


Two new functions were added to database interface. They are more
related to caching server and not authoritative. Add just null pointers,
returning not supporter error if used.

---

diff --git a/src/ldap_driver.c b/src/ldap_driver.c
index 2f6574e..c524b7c 100644
--- a/src/ldap_driver.c
+++ b/src/ldap_driver.c
@@ -959,6 +959,10 @@ static dns_dbmethods_t ldapdb_methods = {
 	setservestalettl,
 	getservestalettl,
 #endif
+#if LIBDNS_VERSION_MAJOR >= 1609
+	NULL, /* setservestalerefresh */
+	NULL, /* getservestalerefresh */
+#endif
 #if LIBDNS_VERSION_MAJOR >= 1600
 	NULL, /* setgluecachestats */
 #endif
